schema.graphql

```diff
@@ -419,6 +420,43 @@ type TransferManager @entity {
   updatedBlock: Block!
 }
 
+type StatType @entity {
+  id: ID! # ticker/op/type/claimIssuer
+  asset: Asset!
+  opType: String!
+  claimType: String
+  claimIssuer: Identity
+  createdBlock: Block!
+  updatedBlock: Block!
+}
+
+type TransferCompliance @entity {
+  id: ID! # ticker/op/type/claimIssuer
+  asset: Asset! @index(unique: false)
+  type: String!
+  statType: StatType!
+  # value is present only in case of MaxInvestorCount and MaxInvestorOwnership
+  value: BigInt
+  # following values are present in case of ClaimCount and ClaimOwnership
+  claimType: String
```

Comment:
Aren't these repeating whats in `StatType`?

Reply:
Originally I didn't have the statType here but when I tried out some transactions apparently we can't delete a statType if transfer compliance is mapped to it, so I added it. I'm OK with removing it, but now I think it would be confusing to consumers to interpret if some values are present in stat type and some in transfer compliance. It already has a confusing flat structure and on top of that having some values come in nested format, I'm not a big fan of it. But said that I can move it if you say so.

Reply:
Makes sense, generally I don't like duplicated data, but probably OK to leave in. Maybe the claim values get grouped in a JSON field? `claim: {value: String!, type: String!, issuer: Identity}`?

Reply:
One problem with the JSON you specified is that we can't use an entity within a JSON. So the issuer field then has to be converted to string